shared/video-sdk/develop/spatial-audio/project-implementation/windows.mdx

```diff
@@ -99,39 +91,32 @@ To implement these features in your <Vpl k="CLIENT" />, take the following steps
 
 1. **Set the spatial position of a remote user**
 
-    To update the spatial position of a remote user, define the `RemoteVoicePositionInfo` and call `updateRemotePosition`. To implement this workflow, in **Dialog Editor**, double-click **Spatial Audio**. **Dialog Editor** automatically creates and opens an event listener for you. Add the following code to the event listener you just created:
+    When a user moves the slider, you create a `RemoteVoicePositionInfo` object and specify information about the position and orientation of the remote user audio stream. Then, you call `updateRemotePosition` with the `RemoteVoicePositionInfo` object and `remoteUId` to update the remote user spatial audio position. To implement this workflow, in **Dialog Editor**, double-click the slider control. **Dialog Editor** automatically set up an event listener for you. Add the following code to the event listener you just created:
 
         ```cpp
+        int value = distnaceBar->GetPos();
```

Comment:
distanceBar undefined.
it should be initialize before using :

CSliderCtrl* distnaceBar = reinterpret_cast<CSliderCtrl*>(GetDlgItem(IDC_SLIDER1));

where  `IDC_SLIDER1` is the diagram id for taken slider.